src/EditorFeatures/Test2/Rename/RenameCommandHandlerTests.vb

```diff
@@ -1251,9 +1238,9 @@ class [|C$$|]
 
                 executeCommand(commandHandler, view, Sub() editorOperations.InsertText(commandInvokedString))
 
-                ' Verify rename session was committed
+                ' Verify rename session was not committed because the scenarios are not supported in async rename.
```

Comment:
a better comment is: if a user does blah blah, then that will cancel rename operation as it is not supported.